packages/tests/package.json

```diff
@@ -18,6 +18,7 @@
     "playwright.config.ts"
   ],
   "dependencies": {
+    "@slashid/slashid": "^3.22.0",
```

Comment:
Added this so we can use the types from here, it's a better DX when writing tests